quality_control_oca_timesheet/readme/USAGE.md

```diff
@@ -0,0 +1,3 @@
+When creating or editing a quality control inspection, you will find a new Timesheets tab within the inspection form. In this tab, you can log and track the time spent on the inspection by adding timesheet entries. Each entry allows you to specify the project, task, date, and the time spent on the inspection in hours and minutes.
+
+When creating or editing a quality control inspection, you will find a new Timesheets tab within the inspection form. In this tab, you can log and track the time spent on the inspection by adding timesheet entries. Each entry allows you to specify the project, task, date, and the time spent on the inspection in hours and minutes.
```

Comment:
There is a line repeated!

Reply:
Ups! Thanks for pointing it out, solved!